src/Frame.js

```diff
@@ -0,0 +1,3 @@
+function Frame() {
+
+};
```

Comment:
Whats this going to be? Seems like you already defined what a frame is in the scorecard js